scripts/lint-dependencies.js

```diff
@@ -0,0 +1,147 @@
+/**
+ * This script finds all the `import` and `require` of a packages
+ * and checks that there's a dependency for each one of them in
+ * the `package.json`.
+ */
+
+const fs = require('fs-extra');
+const path = require('path');
+const globby = require('globby');
+const builtIn = require('builtin-modules');
+
+const typeDependencies = new Set([
+    'har-format',
+    'estree',
+    'request'
+]);
+
+const getPackages = async () => {
+    const pkgs = await globby(['packages/*'], {
+        absolute: true,
+        gitignore: true,
+        onlyFiles: false
+    });
+
+    console.log(`Packages found: ${pkgs.length}`);
+
+    return pkgs;
+};
+
+const getDependencies = (pkg) => {
+    const depArray = [
+        ...builtIn,
+        ...Object.keys(pkg.dependencies || []),
+        ...Object.keys(pkg.devDependencies || []),
+        ...Object.keys(pkg.peerDependencies || []),
+        ...Object.keys(pkg.optionalDependencies || [])
+    ];
+
+    const dependencies = new Set(depArray);
+
+    return dependencies;
+};
+
+const getCodeContent = async (pkgPath) => {
+    const files = await globby([
+        `${pkgPath}/**/*.ts`,
```

Comment:
Planning ahead for once we start using React (may also want `jsx`):
```suggestion
        `${pkgPath}/**/*.ts`,
        `${pkgPath}/**/*.tsx`,
```

Reply:
I'll add `tsx` and `jsx`.